### PR TITLE
docs: update method to disable ESM messages in motd

### PR DESCRIPTION
**Pull request by renanrodrigo**
We will use a marker file rather than an option to the config file.

### CODE REVIEW

File: docs/explanations/motd_messages.md [hunk 1]
```diff
@@ -196,26 +196,16 @@ sudo apt install --reinstall -o Dpkg::Options::="--force-confask" ubuntu-advanta
...
-```sh
-APT::Update::Post-Invoke-Success {"/usr/lib/update-notifier/update-motd-updates-available 2>/dev/null || true";};
-```
-
-to:
-
-```sh
-APT::Update::Post-Invoke-Success {"/usr/lib/update-notifier/update-motd-updates-available --no-esm-messages 2>/dev/null || true";};
-```
+updates, create a file named `/var/lib/ubuntu-advantage/hide-esm-in-motd` file.
```
Comment (s-makin) [inline docs/explanations/motd_messages.md:199]:
```suggestion
updates, create a file named `/var/lib/ubuntu-advantage/hide-esm-in-motd`.
```

Comment (s-makin) [reply docs/explanations/motd_messages.md:199]: (the word file is repeated)

File: docs/explanations/motd_messages.md [hunk 2]
```diff
@@ -196,26 +196,16 @@ sudo apt install --reinstall -o Dpkg::Options::="--force-confask" ubuntu-advanta
...
+There is no need for any content in the file - it's existence is enough to
```
Comment (s-makin) [inline docs/explanations/motd_messages.md:200]:
```suggestion
There is no need for any content in the file - its existence is enough to
```

**Review by s-makin — COMMENTED** [review]
Just two little nits, otherwise lgtm :)

File: docs/explanations/motd_messages.md (hunk 2 above)
Comment (renanrodrigo) [reply docs/explanations/motd_messages.md:200]: me no english good

**Comment by renanrodrigo** [comment]
Addressed @s-makin comments and rebased to the new rst version of docs-devel

File: docs/explanations/motd_messages.rst [hunk 3]
```diff
@@ -220,27 +220,17 @@ Source: MOTD about available updates
...
 Keep in mind that the change in the MOTD message may take some time - if you
-want to remove the ESM related messages immediately, run the script yourself
-using `--force `:
+want to remove the ESM related messages immediately, create the file and run
+the script which generates the messages using `--force `:
 
 .. code-block:: bash
 
-$ sudo /usr/lib/update-notifier/update-motd-updates-available --no-esm-messages --force
+$ sudo /usr/lib/update-notifier/update-motd-updates-available --force
```
Comment (s-makin) [inline docs/explanations/motd_messages.rst:233]:
```suggestion
    $ sudo /usr/lib/update-notifier/update-motd-updates-available --force
```

Comment (s-makin) [reply docs/explanations/motd_messages.rst:233]: Missing indentation was causing the doc checks to fail

Comment (renanrodrigo) [reply docs/explanations/motd_messages.rst]: FIXT

File: docs/explanations/motd_messages.rst [hunk 4]
```diff
@@ -220,27 +220,17 @@ Source: MOTD about available updates
...
-    APT::Update::Post-Invoke-Success {"/usr/lib/update-notifier/update-motd-updates-available --no-esm-messages 2>/dev/null || true";};
+updates, create a file named ``/var/lib/update-notifier/hide-esm-in-motd``.
+There is no need for any content in the file - its existence is enough to
+tell update-notifier, and the Pro Client, to suppress the messages.
 
 Keep in mind that the change in the MOTD message may take some time - if you
-want to remove the ESM related messages immediately, run the script yourself
-using `--force `:
+want to remove the ESM related messages immediately, create the file and run
+the script which generates the messages using `--force `:
```
Comment (s-makin) [inline docs/explanations/motd_messages.rst:229]:
```suggestion
the script which generates the messages using ``--force``:
```

**Review by s-makin — APPROVED** [review]

**Pull request merged**